[2025-07-30T00:31:42.046+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-30T00:31:42.070+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup_resources manual__2025-07-30T00:23:57.754749+00:00 [queued]>
[2025-07-30T00:31:42.082+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup_resources manual__2025-07-30T00:23:57.754749+00:00 [queued]>
[2025-07-30T00:31:42.082+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-30T00:31:42.095+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): cleanup_resources> on 2025-07-30 00:23:57.754749+00:00
[2025-07-30T00:31:42.101+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=113) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-30T00:31:42.103+0000] {standard_task_runner.py:72} INFO - Started process 114 to run task
[2025-07-30T00:31:42.104+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_training', 'cleanup_resources', 'manual__2025-07-30T00:23:57.754749+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmpo_c0rthk']
[2025-07-30T00:31:42.105+0000] {standard_task_runner.py:105} INFO - Job 15: Subtask cleanup_resources
[2025-07-30T00:31:42.144+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_training.cleanup_resources manual__2025-07-30T00:23:57.754749+00:00 [running]> on host 6d8adc46627b
[2025-07-30T00:31:42.206+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kanewyp' AIRFLOW_CTX_DAG_ID='fraud_detection_training' AIRFLOW_CTX_TASK_ID='cleanup_resources' AIRFLOW_CTX_EXECUTION_DATE='2025-07-30T00:23:57.754749+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-30T00:23:57.754749+00:00'
[2025-07-30T00:31:42.207+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-30T00:31:42.216+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-07-30T00:31:42.217+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'rm -f /app/tmp/*.pkl']
[2025-07-30T00:31:42.224+0000] {subprocess.py:99} INFO - Output:
[2025-07-30T00:31:42.227+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-07-30T00:31:42.247+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-30T00:31:42.247+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_training, task_id=cleanup_resources, run_id=manual__2025-07-30T00:23:57.754749+00:00, execution_date=20250730T002357, start_date=20250730T003142, end_date=20250730T003142
[2025-07-30T00:31:42.278+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-30T00:31:42.290+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-30T00:31:42.293+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
